WordPress/Classes/ViewRelated/Aztec/Stock Photos/StockPhotosPlaceholder.swift

```diff
@@ -1,5 +1,9 @@
 // Empty state for Stock Photos
 final class StockPhotosPlaceholder: WPNoResultsView {
+
+    private let companyUrl = "https://www.pexels.com"
```

Comment:
Ideally when there are multiple constants they are collected in an `enum` or `struct`. Not a huge deal, but it's nice.